src/pubtools/_marketplacesvm/tasks/delete/command.py

```diff
@@ -143,10 +143,16 @@ def _delete(
         push_item: VMIPushItem,
         **kwargs,
     ) -> VMIPushItem:
-        marketplaces = self._convert_provider_name(push_item.marketplace_entity_type)
+        # We only support two hyperscalers (Azure, AWS)
+        if push_item.marketplace_entity_type:
```

Comment:
can we have `isinstance(push_item, AmiPushItem)` instead? I believe it would be more reliable.

Reply:
+1 for checking AmiPushItem

and also VHDPushItem instead of fallback everything to else.

else should be use for reporting unsupported pushitem